bin/simple-local-server.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+brew services start elasticsearch
+brew services start redis
+brew services start mysql
+
+yq -yi '.development.host="127.0.0.1" | .test.host="127.0.0.1"' config/database.yml
+sed -i -e 's/redis:6379/localhost:6379/' config/initializers/redis.rb
```

Comment:
Maybe define env variable `REDISCLOUD_URL='redis://localhost:6379'` and not change code?

Reply:
If cache is needed we need to set redis also on `config/environments/development.rb`